proto/observer/genesis.proto

```diff
@@ -18,4 +18,5 @@ message GenesisState {
   PermissionFlags permissionFlags = 4;
   Params params = 5;
   Keygen keygen = 6;
+  LastObserverCount lastObserverCount = 7;
```

Comment:
i would do last_observer_count, permission_flags, 
let's use lowercase underscore for protobuf fields, 
let's try to follow protobuf style guide:
https://protobuf.dev/programming-guides/style/